docusaurus.config.js

```diff
@@ -41,6 +41,8 @@ const config = {
   themeConfig:
     /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
     ({
+      metadata: [{ name: 'keywords', content: 'spicetify, spotify, customization, documentation, blog' }],
```

Comment:
Sorry, but I cannot be able to find this in the [docusaurus API](https://docusaurus.io/docs/api/docusaurus-config).

Reply:
I found it here: https://docusaurus.io/docs/next/seo#global-metadata, and it did work when running it locally.

Reply:
Oh, cool! In that case, thanks!